tests/ctl/core/test_api.py

```diff
@@ -2542,9 +2542,7 @@ def test_system_delete_as_system_manager(
             resource_id=system.fides_key,
             headers=auth_header,
         )
-        assert result.status_code == HTTP_200_OK
-        assert result.json()["message"] == "resource deleted"
-        assert result.json()["resource"]["fides_key"] == system.fides_key
+        assert result.status_code == HTTP_403_FORBIDDEN
```

Comment:
We have to tests when we're making permissions updates, but luckily this already existed!

Reply:
Ah, I missed this, thanks for updating!